config/_default/config.toml

```diff
@@ -86,6 +86,9 @@ breadcrumb_disable = false
 #  Set to false if you don't want to display a logo (/assets/icons/logo.svg) in the top nav bar
 navbar_logo = true
 
+[markup.goldmark.renderer]
+unsafe= true
```

Comment:
Please explain this new setting.

Reply:
This is required for the /docs/reference/api-docs/ to work (because it contains html). This is also the reason why tests fail for https://github.com/cert-manager/website/pull/508 but not for this PR.

Reply:
Please drop a comment in to the code so that we don't forget the motivation for this setting.